python-sdk/langwatch/src/langwatch/__init__.py

```
@@ -0,0 +1,79 @@
+
+
+dspy_available = False
+try:
+    import dspy
+
+    dspy_available = True
+except AttributeError as err:
+    import pydantic
+
+    if pydantic.__version__.startswith("1."):
+        warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)